Pod/Cache/GenericCoreDataStack.swift

```diff
@@ -258,7 +258,7 @@ public class GenericCoreDataStack<CoordinatorType: NSPersistentStoreCoordinator,
             //
             if context.parentContext == mainContext {
                 
-                mainContext.performBlockAndWait({ () -> Void in
+                mainContext.performBlock( { () -> Void in
                     do {
                         try self.mainContext.save()
```

Comment:
Not strictly needed.  The likelihood that there are no changes is low accept in the case where the developer has an edit context and saves for no reason.  Also, I'm limiting non essential changes.  This PR is to change the init to a throwing init in GenericCoreDataStack.